src/gleam_elli_native.erl

```diff
@@ -19,3 +43,6 @@ get_host(Request) ->
         undefined -> <<>>;
         Host when is_binary(Host) -> Host
     end.
+
+mask(Request) ->
+    Request#req{headers = [], body = <<>>}.
```

Comment:
I noticed that newer Elli versions include an `original_headers` field (and in between versions have `parsed_headers`). Perhaps this masking should be inverted to make a new `#req{}` and only fill in the details one would like?

https://github.com/elli-lib/elli/commit/b8f20a970e7e4b20887b8e7edda474314f8c36b6

Reply:
Oh yes! Inverting it sounds wise.